content/03.results-and-discussion.md

```diff
@@ -142,19 +140,19 @@ determining functional differences between ancient bacterial strains (as in
 [@doi:10.1016/j.cub.2017.10.025]). When using a human reference genome,
 nf-core/eager can now also can give estimates of the biological sex of a given
 individual with Sex.DetEERRmine [@doi:10.1038/s41467-018-07483-5]. A dedicated
-'endogenous DNA' calculator (endorS.py) is also included to provide a percentage
-estimate of the sequenced reads matching the reference from the total number of
-reads sequenced per library.
+'endogenous DNA' calculator (endorS.py) is also included to provide an estimate 
+of the percentage of the total sequenced reads per library that map to the provided
```

Comment:
I think this is less understandable, it was Wolfgang who re-wrote this as I thought we need to describe in very basic words what endogenous DNA means (as it's not a common term). @ZandraFagernas @MaxUlysse what do you think what do you think?

Reply:
Happy with either.

Reply:
Lets go with the original. Can you revert?